test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/013_SubscribeVehicleData_to_deprecated_fuelLevel_and_fuelLevel_State.lua

```diff
@@ -0,0 +1,28 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully process SubscribeVehicleData with deprecated fuelLevel and fuelLevel_State
```

Comment:
But SubscribeVehicleDatahas already checked in previous scripts, so propose to remove this script.
In my opinion this is overhead.